docs/configure-amazon-role.md

```diff
@@ -38,13 +38,23 @@ The policy JSON to use in step above:
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Sid": "RedHatProvisioningFederation",
+            "Sid": "AssumeRole",
             "Effect": "Allow",
             "Action": [
                 "sts:AssumeRole",
                 "sts:GetFederationToken"
             ],
             "Resource": "arn:aws:iam::*:role/*"
+        },
+        {
+            "Sid": "DescribeGenericData",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeInstanceTypeOfferings",
+                "ec2:DescribeRegions",
+                "ec2:DescribeInstanceTypes"
+            ],
+            "Resource": "*"
```

Comment:
shall we split this off to a second policy? I believe this is only necessary for the development accounts that do want to refetch the instance types, right?

Reply:
Perhaps it's a good practice overall, but here there is no risk. Let me split this sure.

Reply:
You know what, after I edited the file, it feels a lot more complex, you cannot simply copy and paste a single JSON anymore. Let's keep that in mind and if there are more dangerous permissions added, let's split this. What you think?